pkg/edgeview/README.md

```diff
@@ -34,7 +34,7 @@ edge-view-query [ -token <session-token> ] [ -debug ] [ -inst <instance-id> ] <q
  options:
```

Comment:
Would be great to explain in commit message why `zfsmamanger` paubsub is required here

Reply:
Good point, added